include/net/ieee802154_radio.h

```diff
@@ -171,10 +171,11 @@ enum ieee802154_config_type {
 	 */
 	IEEE802154_CONFIG_CSL_RX_TIME,
 
-	/** Enable/disable or update Enhanced-ACK Based Probing in radio
-	 *  for a specific Initiator.
+	/** Indicates whether to inject IE into ENH ACK Frame for specific address
+	 *  or not. Disabling the ENH ACK with no address provided (NULL pointer)
+	 *  should disable it for all enabled addresses.
 	 */
-	IEEE802154_CONFIG_ENH_ACK_PROBING,
+	IEEE802154_CONFIG_ENH_ACK_IE,
```

Comment:
I would clarify that this configuration option regards Header IE. It is possible to append Payload IEs to Enh-Acks as well.
```suggestion
	IEEE802154_CONFIG_ENH_ACK_HEADER_IE,
```

Reply:
I assumed that this configuration can be used to add Payload IEs to Enh-Acks as well, can't it be?

Reply:
The driver must be aware if the higher layer sets Header or Payload IEs. Additional information is needed, like `_HEADER_` here in config type name, or enumerated field inside `ack_ie` structure. If the new field is preferred, we can add it later when support for Payload IEs is added. However, I would document the structure that it supports only Header IEs right now.